cdap-api/src/main/java/co/cask/cdap/api/metadata/MetadataEntity.java

```diff
@@ -171,12 +252,9 @@ public MetadataEntity appendAsType(String key, String value) {
     }
     List<MetadataEntity.KeyValue> subParts = new ArrayList<>();
     for (KeyValue keyValue : this) {
-      if (!keyValue.getKey().equalsIgnoreCase(splitKey)) {
```

Comment:
should this method really be called head() or prefix()? Where is it used?

Reply:
renamed to head()